src/watcher.js

```diff
@@ -60,12 +64,12 @@ async function runMain({ sendToQueue }) {
 
 async function getLastProcessedBlock() {
   const result = await redis.get(lastBlockRedisKey)
-  lastProcessedBlock = result ? Number(result) : lastProcessedBlock
+  lastProcessedBlock = result ? toBN(result) : lastProcessedBlock
 }
 
 function updateLastProcessedBlock(lastBlockNumber) {
   lastProcessedBlock = lastBlockNumber
-  return redis.set(lastBlockRedisKey, lastProcessedBlock)
+  return redis.set(lastBlockRedisKey, lastProcessedBlock.toString())
```

Comment:
if I call `reset-lastBlock.sh` in CLI which type will be used to write the value to the database?

Reply:
`reset-lastBlock.sh` updates the last block processed in redis. So callingit with `11` is the same as setting `START_BLOCK` to 10 and clearing the DB. Does that make sense?

Reply:
I meant which type of the value will be stored there: number of string?

Reply:
Oh, sorry. All values in redis are strings as far as I know. When we obtain the value from redis, we convert it to BN.

Reply:
ok. thanks